### PR TITLE
Changed some of the texts in prompts.json to a more natural English.

### PR DESCRIPTION
**Pull request by castillobg**
Deleted some unnecessary apostrophes, and (hope to have) improved the overall English used in the prompted sentences.

### CODE REVIEW

File: app/prompts.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   {
     "type": "checkbox",
     "name": "angularModules",
-    "message": "Which Angular's modules would you want to have? (ngRoute and ngResource will be addressed after)",
+    "message": "What Angular modules would you like to have? (ngRoute and ngResource will be addressed after)",
```
Comment (castillobg) [inline app/prompts.json]: Removed unnecessary apostrophe.

File: app/prompts.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
   {
     "type": "list",
     "name": "jQuery",
-    "message": "Would you need jQuery or perhaps Zepto?",
+    "message": "Do you need jQuery or perhaps Zepto?",
```
Comment (castillobg) [inline app/prompts.json]: More idiomatic way of asking.

File: app/prompts.json [hunk 3]
```diff
@@ -115,7 +115,7 @@
   {
     "type": "list",
     "name": "router",
-    "message": "Would you like to use a router ?",
+    "message": "Would you like to use a router?",
```
Comment (castillobg) [inline app/prompts.json]: Removed unnecessary space.

File: app/prompts.json [hunk 4]
```diff
@@ -178,7 +178,7 @@
   {
     "type": "list",
     "name": "bootstrapComponents",
-    "message": "How do you want to implements your Bootstrap components?",
+    "message": "How do you want to implement your Bootstrap components?",
```
Comment (castillobg) [inline app/prompts.json]: Removed 's'.

File: app/prompts.json [hunk 5]
```diff
@@ -213,7 +213,7 @@
   {
     "type": "list",
     "name": "foundationComponents",
-    "message": "How do you want to implements your Foundation components?",
+    "message": "How do you want to implement your Foundation components?",
```
Comment (castillobg) [inline app/prompts.json]: Removed 's'.

File: app/prompts.json [hunk 6]
```diff
@@ -330,7 +330,7 @@
   {
     "type": "list",
     "name": "htmlPreprocessor",
-    "message": "Which html template engine would you want?",
+    "message": "Which HTML template engine would you want?",
```
Comment (castillobg) [inline app/prompts.json]: Changed 'html' to a more standard 'HTML'.

**Comment by Swiip** [comment]
Thanks! English is not our first langage as you can see...

**Comment by castillobg** [comment]
Haha! No problem. Keep it up!

**Pull request merged**